x-pack/legacy/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -56,7 +63,7 @@ export function removeUndefinedProps<T>(obj: T): Partial<T> {
   return pick(obj, value => value !== undefined);
 }
 
-export function getPathParams(pathname: string = '') {
+export function getPathParams(pathname: string = ''): PathParams {
```

Comment:
How I wish we didn't have this method :D
Not sure the best way to get rid of it right now though. Perhaps lift some of this to the routing layer 🤔

Reply:
Not needed for this PR, though!